src/properties.js

```diff
@@ -0,0 +1,137 @@
+
+var target = {
+    tidepool: 'tidepool',
+    diy: 'diy',
+};
+
+var unit = {
+    mmoll: 'mmol/L',
+    mgdl: 'mg/dL',
+};
+
+var screenName = {
+    settings: 'settings',
+    home: 'home',
+    bolus: 'bolus',
+    carbEntry: 'carbEntry',
+};
+
+/**
+ * @summary insulin activity model
+ * @example InsulinModel.Fiasp
+ */
+var settingInsulinModel = {
+    Walsh: 'Walsh',
+    RapidAdults: 'Rapid-Acting – Adults',
+    RapidChildren: 'Rapid-Acting – Children',
+    Fiasp: 'Fiasp'
+};
+
+/**
+ * @summary CGMModel that can be applied to the simulator
+ */
+var settingCGMModel = {
+    Constant: 'Constant',
+    SineCurve: 'Sine Curve',
+    None: 'No Data'
+};
+
+/**
+ * @summary Defaults that can be used to apply to all settings
+ * @example await settings.Apply(SettingDefault)
+ */
+var settingDefault = {
+    /**
+     * @summary DeliveryLimits: { maxBolus: '10.0', maxBasalRate: '3.0' }
+     */
+    DeliveryLimits: { maxBolus: '10.0', maxBasalRate: '3.0' },
+    /**
+     * @summary BasalRates: [{ time: '12:00 AM', unitsPerHour: '0.1' }]
+     */
+    BasalRates: [{ time: '12:00 AM', unitsPerHour: '0.1' }],
+    /**
+     * @summary SuspendThreshold: { value: '75' }
+     */
+    SuspendThreshold: { value: '75' },
+    /**
+     * @summary InsulinModel: InsulinModel.RapidChildren
+     */
+    InsulinModel: settingInsulinModel.RapidChildren,
+    /**
+     * @summary CarbRatios: [{ time: '12:00 AM', carbGramsPerInsulinUnit: '8' }]
+     */
+    CarbRatios: [{ time: '12:00 AM', carbGramsPerInsulinUnit: '8' }],
+    /**
+     * @summary InsulinSensitivities: [{ time: '12:00 AM', bgValuePerInsulinUnit: '500' }]
+     */
+    InsulinSensitivities: [{ time: '12:00 AM', bgValuePerInsulinUnit: '500' }],
```

Comment:
For all of these settings, consider using values that might be reasonable for an average PwD (however you define "average").

Reply:
Yeah I spoke with Shawn about this, for the risk assessment they have a 'average' users settings that I plan to use